### PR TITLE
Imageuploadfeature

### PR DESCRIPTION
**Pull request by KaleyKrim**
users can upload images, image column in tables contain string of path to image

### CODE REVIEW

File: src/containers/NewItemForm/index.js [hunk 1]
```diff
@@ -60,9 +56,20 @@ class NewItemForm extends Component {
...
 
   handleChangeImage(event){
-    this.setState({
-      image: event.target.value
-    })
+    event.preventDefault();
+    let reader = new FileReader();
+
+    let file = event.target.files[0];
+    console.log(file);
```
Comment (jesse-copeland) [inline src/containers/NewItemForm/index.js]: remove console.log() when no longer needed.

File: server/routes/items.js [hunk 2]
```diff
@@ -8,8 +8,17 @@ const route                   = express.Router();
 const db                      = require('../models');
 const {item}                 = db;
 
+
 const multer = require('multer');
-const upload = multer({ dest: 'uploads/' });
+const storage = multer.diskStorage({
+  destination: './uploads',
```
Comment (jesse-copeland) [inline server/routes/items.js:14]: Will you add some error handling later.

File: server/routes/items.js [hunk 3]
```diff
@@ -48,10 +57,12 @@ route.get('/:id', ( req, res ) => {
   });
 });
 
-route.post('/', upload.single('image'), ( req, res ) => {
+route.post('/', upload.single('file'), ( req, res ) => {
   let value = req.isAuthenticated();
   console.log('Is the current user authenticated: ', (value ? 'Yes Baseem' : 'No Baseem'));
   console.log('items route has been requested: POST ');
+  let file = req.file;
+  console.log('file', file);
```
Comment (jesse-copeland) [inline server/routes/items.js]: remove console.log

File: server/routes/items.js [hunk 4]
```diff
@@ -63,7 +74,7 @@ route.post('/', upload.single('image'), ( req, res ) => {
     model : req.body.model,
     dimensions    : req.body.dimensions,
     notes : req.body.notes,
-    image    : req.body.image,
+    image    : req.file.path,
```
Comment (jesse-copeland) [inline server/routes/items.js:76]: Is this uploading the actual data of the image, or is it just the path to the location of the file? You can not reference the image that was selected on a client. You need to pass the image data.

**Comment by ba5eem** [comment]
[Update models - to ensure it support photos URL](https://trello.com/c/fdXcNXjh/47-update-models-to-ensure-it-support-photos-url)

**Pull request merged**